CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/svg/ExportToSVGPlugin.java

```diff
@@ -208,18 +212,16 @@ protected void read(final GraphReadMethods graph, final PluginInteraction intera
             try {
                 Files.delete(file.toPath());
             } catch (final IOException ex) {
-                //To Do: when does this get thrwn?
-                Exceptions.printStackTrace(ex);
+                LOGGER.log(Level.INFO, String.format("Deletion of file %s was not successful", file.toPath()));
```

Comment:
I think this makes more sense as a `WARNING` given something unexpected has happened